src/App.tsx

```diff
@@ -1,34 +1,58 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { useAppDispatch, useAppSelector } from './app/hooks';
+import { actions } from './features/todos';
 
 export const App: React.FC = () => {
+  const [loading, setLoading] = useState(true);
+  const currentTodo = useAppSelector(state => state.currentTodo);
+  const todos = useAppSelector(state => state.todos);
+  const dispatch = useAppDispatch();
+
+  useEffect(() => {
+    setLoading(true);
```

Comment:
you set initial state of loading 'true' on 15 line, and here you set 'true' again. It is better to set 'false' as initial and then change it depend on real loading status.